Assignment-5/task-2.js

```diff
@@ -0,0 +1,19 @@
+// Write a program to sort an array of object on the basis of age ({name:"John", age:26})
+
+const studentObj = [
+  { name: "Sneha", age: 22 },
+  { name: "Parth", age: 20 },
+  { name: "Pritam", age: 23 },
+  { name: "Raksha", age: 21 },
+];
+
+for (let keys in studentObj) {
```

Comment:
don't need to use for in, you can directly use sortedData = studentObj.sort(function (a, b) {
return a.age - b.age;
}); it will give you required output.